_posts/2017-3-26-armory-and-hard-forks.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: Armory and Hard Forks
+category: news
+date: 2017-03-26 23:30:00 -0500
+---
+
+The Armory developers (goatpig, droark, and achow101) oppose hard forks that may attack the original chain. The Armory developers support Segregated Witness and Bitcoin Core. Should a long-lasting hard fork that does not attack the original chain exist, we would consider implementing functionality required to allow Armory users to transact on that chain only if absolutely necessary (e.g. the transaction format for that chain is changed). Depending on how drastic the changes are, we may implement the change fully, implement a migration tool to another wallet that supports the change, or simply not support the fork at all.
```

Comment:
Almost there. I'd reword things slightly. (Missed this the first time. Sorry!) Opening "cold" is a bit strange. I've tossed in a quick intro and am going from there.

"Recently, there has been much debate in the Bitcoin community regarding Segregated Witness, soft forks vs. hard forks, and the usage of alternative miners and nodes. The Armory developers (goatpig, droark, and achow101) fully support Segregated Witness in its current soft fork state, and also support Bitcoin Core. The Armory developers also oppose hard forks that may attack the original chain."